docs/en/apm-server/integrations-index.asciidoc

```diff
@@ -1,91 +1,8 @@
-include::./version.asciidoc[]
-include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
+:doctype: book
 
-:apm-integration-docs:
-:obs-repo-dir:           {observability-docs-root}/docs/en
-:tab-widget-dir:         {docdir}/tab-widgets
-
-:github_repo_link: https://github.com/elastic/apm-server/blob/v{version}
-ifeval::["{version}" == "8.0.0"]
-:github_repo_link: https://github.com/elastic/apm-server/blob/main
-endif::[]
-
-
-// OTHER ATTRS
-// TODO: Check that these are still relevant
-:version: {apm_server_version}
-:beatname_lc: apm-server
-:beatname_uc: APM Server
-:beatname_pkg: {beatname_lc}
-:beat_kib_app: APM app
-:beat_monitoring_user: apm_system
-:beat_monitoring_user_version: 6.5.0
-:beat_monitoring_version: 6.5
-:beat_default_index_prefix: apm
-:access_role: {beat_default_index_prefix}_user
-:beat_version_key: observer.version
-:dockerimage: docker.elastic.co/apm/{beatname_lc}:{version}
-:dockergithub: https://github.com/elastic/apm-server-docker/tree/{doc-branch}
-:dockerconfig: https://raw.githubusercontent.com/elastic/apm-server/{doc-branch}/apm-server.docker.yml
-:discuss_forum: apm
-:github_repo_name: apm-server
-:sample_date_0: 2019.10.20
-:sample_date_1: 2019.10.21
-:sample_date_2: 2019.10.22
-:repo: apm-server
-:no_kibana:
-:no_ilm:
-:no-pipeline:
-:no-processors:
-:no-indices-rules:
-:no_dashboards:
-:apm-server:
-:deb_os:
-:rpm_os:
-:mac_os:
-:docker_platform:
-:win_os:
-:linux_os:
-
-:downloads: https://artifacts.elastic.co/downloads/apm-server
-
-// END OTHER ATTRS
-
-[[apm-user-guide]]
 = APM User Guide
 
-include::apm-overview.asciidoc[]
-
-include::apm-quick-start.asciidoc[]
-
-include::{docdir}/getting-started-apm-server.asciidoc[]
-
-include::data-model.asciidoc[]
-
-include::features.asciidoc[]
-
-include::how-to.asciidoc[]
-
-include::open-telemetry.asciidoc[]
-
-include::manage-storage.asciidoc[]
-
-include::configure/index.asciidoc[leveloffset=+1]
-
-include::{docdir}/setting-up-and-running.asciidoc[]
-
-include::secure-comms.asciidoc[]
-
-include::monitor-apm-server.asciidoc[]
-
-include::api.asciidoc[]
-
-include::troubleshoot-apm.asciidoc[]
-
-include::upgrading.asciidoc[]
-
-include::release-notes.asciidoc[leveloffset=+1]
-
-include::known-issues.asciidoc[leveloffset=+1]
+include::{docs-root}/shared/versions/stack/{source_branch}.asciidoc[]
+include::{docs-root}/shared/attributes.asciidoc[]
```

Comment:
Let's try with `{asciidoc-dir}` instead of `{docs-root}`.

```suggestion
include::{asciidoc-dir}/../../shared/versions/stack/{source_branch}.asciidoc[]
include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
```

Reply:
Note: Attempting this fix because `docs-root` isn't resolving:

```
15:16:53 INFO:build_docs:asciidoctor: ERROR: integrations-index.asciidoc: line 5: include file not found: /tmp/docsbuild/RAyrXFD3Zm/observability-docs/docs/en/apm-server/{docs-root}/shared/versions/stack/main.asciidoc
15:16:53 INFO:build_docs:asciidoctor: ERROR: integrations-index.asciidoc: line 6: include file not found: /tmp/docsbuild/RAyrXFD3Zm/observability-docs/docs/en/apm-server/{docs-root}/shared/attributes.asciidoc
```